packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -132,6 +178,15 @@ export class VaAccordion {
   render() {
     return (
       <Host>
+        {!this.openSingle && (
+          <button
+            ref={el => (this.expandCollapseBtn = el as HTMLButtonElement)}
+            onClick={() => this.expandCollapseAll(!this.expanded)}
+            aria-label="Expand all accordions"
+          >
+            Expand all +
```

Comment:
These could be ternaries relying on `this.expanded`, or we could assign some variables up above the return and just use them here.